xc/xc7/tests/soc/litex/linux/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+set(LITEX_THIRD_PARTY ${symbiflow-arch-defs_SOURCE_DIR}/third_party/litex)
+add_file_target(FILE arty_clocks.xdc)
+
+set(SOURCES
+  top.v
+  top.xdc
+  mem.init
+  mem_1.init
+  mem_2.init
+)
+
+add_litex_test(
+  NAME linux_arty
+  LITEX_THIRD_PARTY ${LITEX_THIRD_PARTY}
+  LITEX_DIR arty_linux_soc
+  LITEX_BOARD a7-35
+  LITEX_SOURCES ${SOURCES}
+  EXTERNAL_SOURCES ${VEXRISCV_LINUX}
+  BOARD arty-full
+  GENERATE_SCRIPT ${GENERATE_LINUX_LITEX}
+  FIXUP_SCRIPT ${FIXUP_XDC}
+  FLAGS
```

Comment:
I think if there are no flags, this can be avoided

Reply:
Yes, indeed. Removed.